docker/databricks/requirements_10.4ML.txt

```diff
@@ -0,0 +1,241 @@
+gast==0.4.0
+gitdb==4.0.7
+GitPython==3.1.12
+google-auth==1.22.1
+google-auth-oauthlib==0.4.2
+google-pasta==0.2.0
+grpcio==1.39.0
+gunicorn==20.0.4
+gviz-api==1.10.0
+h5py==3.1.0
+hijri-converter==2.2.3
+holidays==0.12
+horovod==0.23.0
+htmlmin==0.1.12
+huggingface-hub==0.1.2
+idna==2.10
+ImageHash==4.2.1
+imbalanced-learn==0.8.1
+importlib-metadata==3.10.0
+importlib-metadata
+ipykernel==5.3.4
+ipython==7.22.0
+ipython-genutils==0.2.0
+ipywidgets==7.6.3
+isodate==0.6.0
+itsdangerous==1.1.0
+jedi==0.17.2
+Jinja2==2.11.3
+jmespath==0.10.0
+joblib==1.0.1
+joblibspark==0.3.0
+jsonschema==3.2.0
+jupyter-client==6.1.12
+jupyter-core==4.7.1
+jupyterlab-pygments==0.1.2
+jupyterlab-widgets==1.0.0
+keras==2.8.0
+Keras-Preprocessing==1.1.2
+kiwisolver==1.3.1
+koalas==1.8.2
+korean-lunar-calendar==0.2.1
+langcodes==3.3.0
+libclang==13.0.0
+lightgbm==3.3.2
+llvmlite==0.38.0
+LunarCalendar==0.0.9
+Mako==1.1.3
+Markdown==3.3.3
+MarkupSafe==2.0.1
+matplotlib==3.4.2
+missingno==0.5.1
+mistune==0.8.4
+#mleap==0.18.1
+mleap
+mlflow-skinny==1.24.0
+multimethod==1.7
+murmurhash==1.0.5
+nbclient==0.5.3
+nbconvert==6.0.7
+nbformat==5.1.3
+nest-asyncio==1.5.1
+networkx==2.5
+nltk==3.6.1
+notebook==6.3.0
+numba==0.55.1
+numpy==1.20.1
+oauthlib==3.1.0
+opt-einsum==3.3.0
+packaging==21.3
+pandas==1.2.4
+pandas-profiling==3.1.0
+pandocfilters==1.4.3
+paramiko==2.7.2
+parso==0.7.0
+pathy==0.6.0
+patsy==0.5.1
+petastorm==0.11.4
+pexpect==4.8.0
+phik==0.12.0
+pickleshare==0.7.5
+Pillow==8.2.0
+plotly==5.5.0
+pmdarima==1.8.4
+preshed==3.0.5
+prompt-toolkit==3.0.17
+prophet==1.0.1
+protobuf==3.17.2
+psutil==5.8.0
+psycopg2==2.8.5
+ptyprocess==0.7.0
+pyarrow==4.0.0
+pyasn1==0.4.8
+pyasn1-modules==0.2.8
+pybind11==2.9.1
+pycparser==2.20
+pydantic==1.8.2
+Pygments==2.8.1
+PyGObject==3.36.0
+PyMeeus==0.5.11
+PyNaCl==1.4.0
+pyodbc==4.0.30
+pyparsing==2.4.7
+pyrsistent==0.17.3
+pystan==2.19.1.1
+python-apt
+python-dateutil==2.8.1
+python-editor==1.0.4
+python-engineio==4.3.0
+python-socketio==5.4.1
+pytz==2020.5
+PyWavelets==1.1.1
+PyYAML==5.4.1
+pyzmq==20.0.0
+regex==2021.4.4
+requests==2.25.1
+requests-oauthlib==1.3.0
+requests-unixsocket==0.2.0
+rsa==4.7.2
+s3transfer==0.3.7
+sacremoses==0.0.46
+scikit-learn==0.24.1
+scipy==1.6.2
+seaborn==0.11.1
+Send2Trash==1.5.0
+setuptools-git==1.2
+shap==0.40.0
+simplejson==3.17.2
+six==1.15.0
+slicer==0.0.7
+smart-open==5.2.0
+smmap==3.0.5
+spacy==3.2.1
+spacy-legacy==3.0.8
+spacy-loggers==1.0.1
+spark-tensorflow-distributor==1.0.0
+sqlparse==0.4.1
+srsly==2.4.1
+ssh-import-id==5.10
+statsmodels==0.12.2
+tabulate==0.8.7
+tangled-up-in-unicode==0.1.0
+tenacity==6.2.0
+tensorboard==2.8.0
+tensorboard-data-server==0.6.1
+tensorboard-plugin-profile==2.5.0
+tensorboard-plugin-wit==1.8.1
+tensorflow==2.8.0
+tensorflow-estimator==2.8.0
+tensorflow-io-gcs-filesystem==0.24.0
+termcolor==1.1.0
+terminado==0.9.4
+testpath==0.4.4
+tf-estimator-nightly==2.8.0.dev2021122109
+thinc==8.0.12
+threadpoolctl==2.1.0
+tokenizers==0.10.3
+#torch==1.10.2+cu111
```

Comment:
@rwlee Note for the 2 libs, intially they were below version in 10.4 LTS:
```
torch==1.10.2+cu111
torchvision==0.11.3+cu111
```
However when i was testing it, it could not find the version.
So I changed to 
```
torch==1.10.2
torchvision==0.11.3
```

Just FYI that note sure if you want to test if there is way to match exactly the same version as 10.4LTS